src/net/sourceforge/kolmafia/session/BreakfastManager.java

```diff
@@ -882,4 +885,29 @@ private static void useBookOfEverySkill() {
       RequestThread.postRequest(UseItemRequest.getInstance(book));
     }
   }
+
+  private static void useLegacyBooks() {
```

Comment:
Why legacy over replica here and in the prefs?

Reply:
because Legacy of Loathing. Could be replica I guess 🤷

Reply:
I'd like replica just so I can prefref replica, for example.

Reply:
okay, sure